tests/certification/bindings/alicloud/nacos/README.md

```diff
@@ -0,0 +1,18 @@
+# Alicloud Nacos Binding certification testing
```

Comment:
We need a more detailed test plan before this component can be certified.

Please, compare with the test plans we created for other components.